packages/dev/inspector-v2/src/components/booleanProperty.tsx

```diff
@@ -0,0 +1,56 @@
+// eslint-disable-next-line import/no-internal-modules
+import type { Observable } from "core/index";
+
+import type { SwitchProps } from "@fluentui/react-components";
+import type { ChangeEvent, FunctionComponent } from "react";
+
+import { InfoLabel, makeStyles, Switch, tokens } from "@fluentui/react-components";
+import { useCallback, useMemo } from "react";
+import { useObservableState } from "../hooks/observableHooks";
+
+// TODO: This is just an example. Need to converge this with however we want this type of property like component to work across all tools with the UI overhaul.

```

Comment:
So this will get replaced by georgie's work right?

Reply:
Yes you nailed it! This is where @georginahalpern  and I will be collaborating on specific re-usable controls, and why property pane controls are just stubbed out for testing.